bld/CLMBuildNamelist.pm

```diff
@@ -3866,8 +3865,8 @@ sub setup_logic_fates {
 
     if (&value_is_true( $nl_flags->{'use_fates'})  ) {
         add_default($opts, $nl_flags->{'inputdata_rootdir'}, $definition, $defaults, $nl, 'fates_paramfile', 'phys'=>$nl_flags->{'phys'});
-        my @list  = (  "use_fates_spitfire", "use_fates_planthydro", "use_fates_ed_st3", "use_fates_ed_prescribed_phys", 
-                       "use_fates_inventory_init", "use_fates_logging","fates_parteh_mode" );
+        my @list  = (  "fates_spitfire_mode", "use_fates_planthydro", "use_fates_ed_st3", "use_fates_ed_prescribed_phys", 
+                       "use_fates_inventory_init","use_fates_fixed_biogeog", "use_fates_logging","fates_parteh_mode", "use_fates_cohort_age_tracking" );
```

Comment:
Fates_nat_and_anthro_ignitions tests the new option 4 of fates_spitfire_mode. We should add a test for option 3. There already are tests for 0 and 1. It's probably OK to NOT do option 2 since option 4 does it with the addition of another dataset type.

Reply:
So the missing test seems to be for use_fates_fixed_biogeog. Is that mature enough that we can add a FATES SP mode test in?

[ ] Add a FATES-SP mode testmod and test to the testlist.

Reply:
`use_fates_fixed_biogeog` has a test with the long testmod name of `FatesReducedComplexFixedBiogeo`: https://github.com/glemieux/ctsm/tree/fates_main_api-merge-ctsm_master/cime_config/testdefs/testmods_dirs/clm/FatesReducedComplexFixedBiogeo

The `FatesReducedComplex` prefix was used in anticipating of multiple reduced complexity variation, I believe fates-sp mode being among them.  That test is still waiting on completion of #1182 and is in https://github.com/ESCOMP/CTSM/commit/7e5289561ba2fe319b82b034e27c66d26d2657dc.

Reply:
Ahh, OK I missed that one. So this is taken care of.